frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/block_action_add_button.dart

```diff
@@ -21,25 +24,35 @@ class BlockAddButton extends StatelessWidget {
   Widget build(BuildContext context) {
     return BlockActionButton(
       svgName: 'editor/add',
-      richMessage: const TextSpan(
+      richMessage: TextSpan(
         children: [
           TextSpan(
-            // todo: l10n.
-            text: 'Click to add below',
+            text: LocaleKeys.blockActions_addBelowTooltip.tr(),
+          ),
+          TextSpan(
+            text: LocaleKeys.blockActions_addAboveTooltip.tr(),
           ),
         ],
       ),
       onTap: () {
+        final isAltPressed =
+            KeysPressedManager.of(context)?.isAltPressed ?? false;
+
         final transaction = editorState.transaction;
-        // if the current block is not a empty paragraph block, then insert a new block below the current block.
+
+        /// If the current block is not an empty paragraph block,
```

Comment:
nit: triple slash is unnecessary because it's not a documentation comment?

Reply:
Yes well, for any multiline comment, I always do triple slashes, it doesn't really change anything though.

I've gotten used to this because with three slashes my IDE conveniently adds another three slashes on a line break.